src/backlight/strategies/exit.py

```diff
@@ -80,10 +80,8 @@ def _exit(
 
         df = pd.DataFrame(index=indices, data=exits, columns=["amount", "_id"])
 
-        return from_dataframe(df, symbol, currency_unit)
+        return from_dataframe(df, entries.symbol, trades.currency_unit)
```

Comment:
`from_dataframe(df, trades.symbol, trades.currency_unit)`?

Reply:
Oh, yes. Thank you.